src/Directory.Build.props

```diff
@@ -1,6 +1,6 @@
 <Project>
   <PropertyGroup>
-    <Version>0.4</Version>
+    <Version>0.5.0</Version>
```

Comment:
I would also recommend updating PSRule and its baseline file as part of a new release (there are some notes about that on the OneNote), mentioning in case those steps were not considered